domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/CameraXCameraUseCase.kt

```diff
@@ -130,6 +133,7 @@ constructor(
         }
     }
 
+    @RequiresApi(Build.VERSION_CODES.Q)
```

Comment:
Should we consider changing our min API version, if this is needed.

Reply:
I dont think we should change the min API version of our entire project for the traces. The traces are created for and only being used in the project by macrobenchmark. I could just guard the trace calls with an api check instead

Reply:
Guarding the trace calls should work